logging/interceptor.go

```diff
@@ -218,6 +218,11 @@ func addAccountIDField(ctx context.Context, fields logrus.Fields) error {
 func addCustomField(ctx context.Context, fields logrus.Fields, customField string) error {
 	field, err := auth.GetJWTField(ctx, customField, nil)
 	if err != nil {
+		if customField == DefaultSubjectKey {
+			// non client token does not have subject field, ignore
+			return nil
+		}
+
```

Comment:
Updates to this project need PR tests. Please add a unit test verifying what this functionality does